.rubocop.yml

```diff
@@ -1,2 +1,5 @@
 Metrics/LineLength:
   Enabled: false
+
+Style/BlockComments:
```

Comment:
We shouldn't have any comments on the source code, we have git history instead :)